pkg/virt-api/rest/subresource.go

```diff
@@ -401,6 +417,37 @@ func (app *SubresourceAPIApp) RestartVMRequestHandler(request *restful.Request,
 		return
 	}
 
+	if forceRestartBool {
+		pods, err := app.virtCli.CoreV1().Pods(namespace).List(k8smetav1.ListOptions{})
+		if err != nil {
+			response.WriteError(http.StatusForbidden, err)
+			return
+		}
+		if len(pods.Items) == 0 {
+			response.WriteError(http.StatusForbidden, fmt.Errorf("Could not list any pods"))
+			return
+		}
+		var controlledPod *k8sv1.Pod
+		for _, pod := range pods.Items {
+			if controller.IsControlledBy(&pod, vmi) {
+				controlledPod = &pod
+				break
+			}
+		}
+		if controlledPod == nil {
+			response.WriteError(http.StatusForbidden, fmt.Errorf("no controlled pod was found for VMI"))
```

Comment:
What's a "controlled pod" in this context?

Reply:
Added a comment in code